contribs/docker/wazo-auth-mock/wazo-auth-mock.py

```diff
@@ -23,8 +23,12 @@
     url_prefix = ''
 
 DEFAULT_POLICIES = {
+    'wazo_default_master_user_policy': {
+        'uuid': '00000000-0000-4000-0000-000000000001',
+        'name': 'wazo_default_admin_policy',
+    },
     'wazo_default_admin_policy': {
-        'uuid': '00000000-0000-0000-0000-000000000001',
+        'uuid': '00000000-0000-4000-0000-000000000002',
```

Comment:
FYI it still not a valid UUID4
```suggestion
        'uuid': '00000000-0000-4000-[8-9-a-b]000-000000000002',
```

Reply:
True, thank you. I replaced with a random UUID, ending with 0001 and 0002